src/Tribe/Template/Month.php

```diff
@@ -609,6 +609,7 @@ private function get_daily_events( $date ) {
 
 			// this  will skip updating term and meta caches - those were already
 			// updated in $this->set_events_in_month()
+			//Order of Events Sticky Events, All Day Events, then Events by start time
```

Comment:
I'm not sure the comment is accurate - as things stand wouldn't a multiday event (that began before whatever the current day is) trump an all day event? Might be worth simplifying to `'Ordered by start time, sticky events first'` or something like that.